lib/nuts.js

```diff
@@ -250,6 +251,91 @@ Nuts.prototype.onUpdate = function(req, res, next) {
     .fail(next);
 };
 
+Nuts.prototype.onElectronUpdate = function(channel, platform, filetype) {
```

Comment:
Put everything here on a separate file (keep in this file only the routes definition)